README.md

```diff
@@ -1,6 +1,6 @@
 # fsl
 
-FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data, created by the Analysis Group, FMRIB, Oxford, UK.
+FSL is a comprehensive library of analysis tools for FMRI, MRI and DTI brain imaging data, created by the Analysis Group, FMRIB, Oxford, UK. See https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSL for more information.
```

Comment:
Regarding versioning below. there is no integration version to change. You might want to remove that.

Reply:
also there is no `make version-service-*` only `make version-*` is available.

Reply:
Also this is not gitlab, this is GitHub. I would remove the reference to that.

Reply:
Also remove the gitlab reference, it is of no value to external users. I would drop it.

Reply:
Thanks, I've updated the README accordingly. Also added some instructions on how to test the application (mostly for us or whoever wants to deploy a new version).